### ISSUE TITLE
A useful modifier

### ISSUE THREAD

**Issue by Ksc25** [MoreRobustThanYou/Kerbalism#863]
Actually I´m working in some changes to my kerbalism profile and notice that many things could be done just by adding a modifier that return the inverse of the amount of a resource. This way is easier to add more complexity to the behavior of some processes.

**Comment by lordcirth** [comment]
What do you mean by "inverse"? The amount of empty capacity for that resource?

**Comment by Ksc25** [comment]
No, the multiplicative inverse, 1/ResourceAmount, so you can for example obtain a food to water ratio by multplying the food amount by the inverse of the water amount
